HACKING.adoc

```diff
@@ -410,3 +410,21 @@ This is done using the following command:
 
 Files in the `examples/` directory are used as tests.
 Resulting slides are kept in `test/doctest/`.
+
+
+== Netlify Integration
+
+We integrate with https://www.netlify.com[Netlify] in order to automatically convert examples into slides.
```

Comment:
It's true that Netlify run the `publish` Rake task but we are mostly using Netlify to publish static content

Reply:
How about this?

```suggestion
On every commit or PR, the https://www.netlify.com[Netlify] service will convert some examples into slides and host the resulting pages on its platform where it will be visible by anyone.
It hosts the converted HTML files, reveal.js framework and static content like images and CSS.
This integration will allow us to easily preview PRs and demo features to users (source and converted result).
```

Reply:
Perfect 👌